db/migrate/20200129123555_add_disable_option_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddDisableOptionUser < ActiveRecord::Migration[5.2]
```

Comment:
According to the migration naming standards, we should use instead:
```suggestion
class AddDisabledToUsers < ActiveRecord::Migration[5.2]
```